Templates/TimerTrigger-Powershell/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "defaultFunctionName": "TimerTriggerPowershell",
+    "description": "A Powershell function that will be run on a specified schedule",
+    "name": "TimerTrigger - Powershell",
+    "language": "Powershell",
+    "category": [
+        "Experimental",
+        "Data Processing"
```

Comment:
You are right. I need to either remove this from data processing and leave it just in experimental or have the other experimental templates show up in respective categories as well.
@christopheranderson , @mattchenderson  thoughts?